src/SeleniumLibrary/keywords/element.py

```diff
@@ -723,24 +725,91 @@ def simulate(self, locator, event):
 
     @keyword
     def press_key(self, locator, key):
-        r"""Simulates user pressing key on element identified by ``locator``.
-
-        See the `Locating elements` section for details about the locator
-        syntax.
-
-        ``key`` is either a single character, a string, or a numerical ASCII
-        code of the key lead by '\\'.
-
-        Examples:
-        | `Press Key` | text_field   | q     |
-        | `Press Key` | text_field   | abcde |
-        | `Press Key` | login_button | \\13  | # ASCII code for enter key |
-        """
+        """Deprecated use `Press Keys` instead."""
         if key.startswith('\\') and len(key) > 1:
             key = self._map_ascii_key_code_to_key(int(key[1:]))
         element = self.find_element(locator)
         element.send_keys(key)
 
+    @keyword
+    def press_keys(self, locator=None, *keys):
+        """Simulates user pressing key(s) to an element or on the active browser.
+
+
+        If ``locator`` is None or false, then the ``keys`` are sent to the currently
```

Comment:
If locator evaluates as false, see `Boolean arguments`  for more details, then keys are send to currently active browser. Otherwise element is searched and keys are send to the element identified by the locator. In later case keyword fails if element is not found. See the locating... This version could be better